tests/src/Kernel/Access/OgEntityAccessTest.php

```diff
@@ -133,6 +177,15 @@ protected function setUp() {
       ->grantPermission($this->randomMachineName())
       ->save();
 
+    $this->ogAdminRole = OgRole::create();
+    $this->ogAdminRole
+      ->setId($this->randomMachineName())
+      ->setLabel($this->randomString())
+      ->setGroupType($this->group1->getEntityTypeId())
+      ->setGroupBundle($this->groupBundle)
+      ->setIsAdmin(TRUE)
+      ->save();
```

Comment:
Currently not, we have 

```php
  protected function createPerBundleRoles($entity_type_id, $bundle_id) {
    foreach ($this->getDefaultRoles() as $role_name => $default_properties) {
      $properties = [
        'group_type' => $entity_type_id,
        'group_bundle' => $bundle_id,
        'id' => $role_name,
        'role_type' => OgRole::getRoleTypeByName($role_name),
      ];
```

(no `is_admin`). I'll create a follow up issue for this.